lib/src/internal/DomainWatcher.cpp

```diff
@@ -279,6 +279,7 @@ namespace mxl::lib
                 std::lock_guard<std::mutex> lock(_mutex);
                 watchCount = _watches.size();
                 eventsToMonitor.reserve(watchCount);
```

Comment:
There is no point in a reserve immediately before a resize, should have just changed it to the latter. Or more in keeping with the original code, might have been to add a `push_back` before the `op[]` below.